Misc/NEWS.d/next/Library/2022-03-23-10-07-41.bpo-47098.7AN_qp.rst

```diff
@@ -0,0 +1,2 @@
+The Keccak Code Package for :mod:`hashlib`'s SHA3 and SHAKE algorithms has
+been replaced with tiny_sha3.
```

Comment:
clarify that this is only used when OpenSSL does not support the algorithms to avoid freaking anyone out.